From a5f0e20ea3cb857c7a7ae9a6664eb878d64a3db3 Mon Sep 17 00:00:00 2001
From: Andre Guedes <aguedespe@gmail.com>
Date: Wed, 21 Mar 2012 00:03:35 -0300
Subject: [PATCH 0040/1187] Bluetooth: Add Periodic Inquiry command complete
 handler

commit 79d6e068bee82e9998b2be78bc0f08f2dec8777a	upstream

This patch adds a handler function to Periodic Inquiry command
complete event.

Signed-off-by: Andre Guedes <aguedespe@gmail.com>
Acked-by: Marcel Holtmann <marcel@holtmann.org>
Signed-off-by: Gustavo Padovan <gustavo@padovan.org>
Signed-off-by: Lans Zhang <jia.zhang@windriver.com>
---
 include/net/bluetooth/hci.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/net/bluetooth/hci.h b/include/net/bluetooth/hci.h
index 3edb3e7..9cf088d 100644
--- a/include/net/bluetooth/hci.h
+++ b/include/net/bluetooth/hci.h
@@ -324,6 +324,8 @@ struct hci_cp_inquiry {
 
 #define HCI_OP_INQUIRY_CANCEL		0x0402
 
+#define HCI_OP_PERIODIC_INQ		0x0403
+
 #define HCI_OP_EXIT_PERIODIC_INQ	0x0404
 
 #define HCI_OP_CREATE_CONN		0x0405
-- 
1.7.5.4

